Add content for the Third year undergraduate courses

Cydonia01 opened the pull request:
Also, added former tags for academicians.

doganulus commented on data/bouncmpe/people/ozgovde-atay.json:
```diff
@@ -7,7 +7,8 @@
     "cmpe250",
-    "advisor-1"
+    "advisor-1",
+    "former-cmpe350"
```
Do we know Atay hoca taught CMPE350 in the past?

Cydonia01 replied:
![Ekran görüntüsü 2024-10-13 224408](https://github.com/user-attachments/assets/0a762dc6-030c-4b53-8a5b-b858d5472410)
I saw that in website. Also, there is no info before 2010

doganulus commented:
For each course description, let's add a warning on the top for users, saying that 

```
The information on this page is under construction. 
```

Then, I will merge it.

Cydonia01 commented:
I added under construction warning to the 3XX en tr pages.

Mutti499 commented:
I have checked and adjusted my own warning files according to yours. But I think this is not a dynamic solution because the word “warning” could be anything in future problems rather an "under construction" message.

Cydonia01 commented:
Added photos to static/images/people directory, because assets/images directory is not added to the public upon running server. Also changed the card template to make the images square.

doganulus commented:
Let's keep PRs seperate. @Cydonia01 

- Revert your lateet commits on this PR. This PR is only about course content.
- Open a new branch for photos and create a new PR from that branch. 
- The research page will have its own PR, too.

Cydonia01 commented:
I reverted it. Sorry for not notifying before.

doganulus commented:
Thank you @Cydonia01. Can you rebase your Pr to `main` and update warning tags with my language-aware warning shortcode on the `main` branch. It's called `under-construction-warning`.
